### PR TITLE
FT2-109 Add more cards to dashboard widget

### PR DESCRIPTION
**Pull request by lazcool**
Adds Waiting for Review, In Review and Published cards to the dashboard widget.

Also
* fix review type item filtering

### CODE REVIEW

File: DFC.ServiceTaxonomy.ContentApproval/Migrations.cs [hunk 1]
```diff
@@ -115,5 +115,12 @@ public async Task<int> UpdateFrom4Async()
 
             return 5;
         }
+
+        public async Task<int> UpdateFrom5Async()
+        {
+            await _recipeMigrator.ExecuteAsync("stax-content-approval-amendment-03.recipe.json", this);
+
+            return 6;
```
Comment (andycyberian) [inline DFC.ServiceTaxonomy.ContentApproval/Migrations.cs:123]: Can the Create method be updated to return 6?

**Review by andycyberian — COMMENTED** [review]
All looks good, just one comment on the migrations.

**Review by pramireddy — COMMENTED** [review]
Code looks fine for me.

**Comment by pramireddy** [comment]
It's looks good

**Review by pramireddy — APPROVED** [review]
It's look good for me.

**Pull request merged**